peps/pep-0723.rst

```diff
@@ -55,24 +55,6 @@ Rationale
 This PEP defines a mechanism for embedding metadata *within the script itself*,
 and not in an external file.
 
-We choose to follow the latest developments of other modern packaging
-ecosystems (namely `Go`__ and provisionally `Rust`__) by embedding the existing
-file used to describe projects, in our case ``pyproject.toml``.
-
-__ https://github.com/erning/gorun
-__ https://rust-lang.github.io/rfcs/3424-cargo-script.html
-
-The format is intended to bridge the gap between different types of users
-of Python. Users will benefit from seamless interoperability with tools that
-already work with TOML.
-
-One of the central themes we discovered from the recent
-`packaging survey <https://discuss.python.org/t/22420>`__ is that users have
-begun getting frustrated with the lack of unification regarding both tooling
-and specs. Adding yet another metadata format (like :pep:`722` syntax for a
-list of dependencies), even for a currently unsatisfied use case, would
-further fragment the community.
-
```

Comment:
This can likely stay, no?

Reply:
Maybe? I wasn't 100% sure that "adding another format would further fragment the community" was appropriate now that this *is* a (somewhat) different format. But it's marginal and I could be persuaded to leave it - there's certainly a personal bias because I was very against the whole "make it like `pyproject.toml`" idea, so I appreciate people keeping me honest over that.

Reply:
I think it could stay if changed to phrase it more along the lines of that the structure comes from `pyproject.toml`, but the allowed data is deviation. So the use of TOML, `[tool]`, etc. are not a new format, we just introduce some new keys unique to this use case.

Reply:
OK. I can have a go at that. Can't promise it'll be a particularly compelling addition to the rationale, though 😉

Reply:
OK, added. Let me know what you think.

Reply:
LGTM!

Reply:
Also LGTM!